frontend/src/ClassDetail.svelte

```diff
@@ -166,7 +166,7 @@ tr td:not(:nth-of-type(1)):not(:nth-of-type(2)):not(:last-child) {
       {#if showStudentsList}
         {#if clazz.summary }
           <button class="p-0 btn btn-link"
-                    on:click={() => showSummary = !showSummary}>{showSummary ? "Skrýt" : "Zobrazit"} informace o cvičení</button>
+                    on:click={() => showSummary = !showSummary}>{showSummary ? "Hide" : "Show"} exercise summary</button>
```

Comment:
```suggestion
                    on:click={() => showSummary = !showSummary}>{showSummary ? "Hide" : "Show"} class summary</button>
```
We use "class" for this in Kelvin.